cluster/gce/util.sh

```diff
@@ -675,17 +686,40 @@ function kube-up {
   
   create-node-instance-template $template_name
 
+  if [[ ${MAX_INSTANCES_PER_MIG:-500} -le "0" ]]; then
+    echo "MAX_INSTANCES_PER_MIG cannot be negative. Assuming default"
+    MAX_INSTANCES_PER_MIG=500
+  fi
+  local num_migs=$(((${NUM_NODES} + ${MAX_INSTANCES_PER_MIG:-500} - 1) / ${MAX_INSTANCES_PER_MIG:-500}))
+  local instances_per_mig=$((${NUM_NODES} / ${num_migs}))
+  local last_mig_size=$((${NUM_NODES} - (${num_migs} - 1) * ${instances_per_mig}))
+
+  for i in $(seq $((${num_migs} - 1))); do
+    gcloud compute instance-groups managed \
+        create "${NODE_INSTANCE_PREFIX}-group-$i" \
+        --project "${PROJECT}" \
+        --zone "${ZONE}" \
+        --base-instance-name "${NODE_INSTANCE_PREFIX}" \
+        --size "${NUM_NODES}" \
+        --template "$template_name" || true;
+    gcloud compute instance-groups managed wait-until-stable \
```

Comment:
Can you add a TODO to spin these all up in parallel and then do a bunch of waits at the end?